pkgs/applications/audio/spectmorph/default.nix

```
@@ -2,10 +2,10 @@
 
 stdenv.mkDerivation rec {
   pname = "spectmorph";
-  version = "0.5.2";
+  version = "0.6.1";
   src = fetchurl {
-    url = "https://spectmorph.org/files/releases/${pname}-${version}.tar.bz2";
-    sha256 = "0yrq7mknhk096wfsx0q3b6wwa2w5la0rxa113di26rrrw136xl1f";
+    url = "https://github.com/swesterfeld/spectmorph/releases/download/${version}/${pname}-${version}.tar.bz2";
+    sha256 = "sha256-H/PaczAkjxeu2Q6S/jazZ0PU9oCmhBzsLgbGLusxXm8=";
```

The SRI-hash (which you are already using) is the preferred way in 2023. But as an addition, all the fields currently gets merged into `hash`, so we can skip a little evaluation moving to that. (But this is not a blocking change request)

```suggestion
    hash = "sha256-H/PaczAkjxeu2Q6S/jazZ0PU9oCmhBzsLgbGLusxXm8=";
```